modules/tinymce/src/core/main/ts/init/InitIframe.ts

```diff
@@ -83,11 +83,10 @@ const getIframeHtml = (editor: Editor) => {
 };
 
 const createIframe = (editor: Editor, o) => {
-  const title = editor.editorManager.translate(
-    'Rich Text Area. Press ALT-0 for help.'
-  );
+  const iframeTitle = editor.getParam('content_aria_label', 'Rich Text Area. Press ALT-0 for help.');
```

Comment:
This should be in `Settings.ts` not here, as we try to put all settings in the `Settings.ts` file.